### PR TITLE
Movement/Selection Help: fix for v2.5/v2.6

### PR DESCRIPTION
**Pull request by butterw**
This PR attempts to make the Movement/Selection Help display more readable and update it (it fixes incorrect out-of-date info in the v2.5/v2.6.

https://forums.triplea-game.org/topic/2799/make-movement-selection-help-menu-more-readable-hotkeys-discussion-zoom-level-display

### CODE REVIEW

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/MoveHelpMenu.java [hunk 1]
```diff
@@ -20,53 +20,50 @@ Action buildMenu() {
...
-                  + "Left click on a territory but not on a unit to bring up a selection "
-                  + "window for inputting the desired selection.<br>"
-                  + "<br><b> Deselecting Units</b><br>"
-                  + "Right click somewhere not on a unit stack to unselect the last "
-                  + "selected unit.<br>"
-                  + "Right click on a unit stack to unselect one unit in the stack.<br>"
-                  + "ALT-Right click on a unit stack to unselect 10 units of that type in "
-                  + "the stack.<br>"
-                  + "CTRL-Right click on a unit stack to unselect all units of that type "
+              "<font size='+1'>Selecting Units</font><br>"
```
Comment (RoiEXLab) [inline game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/MoveHelpMenu.java:23]: +1 ?
Is this the same like a simple '1'?

**Review by RoiEXLab — APPROVED** [review]
Just curious about the unusual '+1' other than that it looks good

File: game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/MoveHelpMenu.java (hunk 1 above)
Comment (butterw) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/MoveHelpMenu.java:23]: html font size is a number from 1 (small) to 7 (large). +1 means 1 size larger. 
It's deprecated in html5 which uses css.
On my win10 machine with Substance the default font size is '4'.

Comment (RoiEXLab) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/MoveHelpMenu.java:23]: Ah ok thanks for the clarification

Comment (butterw) [reply game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/MoveHelpMenu.java:23]: I should mention the display (JEditorPane) is in a JScrollPane, however the scrolling bars will not be displayed even if the text is larger than the screen as the viewport of JScrollPane is not specified. Not a problem here as the text will fit in screen. The JScrollPane could be removed.

**Pull request merged**